quickwit-pushapi/src/errors.rs

```diff
@@ -31,6 +31,8 @@ pub enum PushApiError {
     QueueAlreadyExists { queue_id: String },
     #[error("PushAPI service is down")]
     PushAPIServiceDown,
+    #[error("The following indexes `{queue_ids:?}` don't exist.")]
+    IndexDoesNotExist { queue_ids: Vec<String> },
```

Comment:
There was already an option called IndexDoesNotExist. Should we rename

QueueDoes(Not)?Exist to IndexDoes(Not)?Exist

Reply:
We can rename but we lose precision between:
- Index missing error (metastore)
- queue missing error (queues)
Are you good with renaming `IndexDoesNotExist` ?

Reply:
I'm good with it

Reply:
Both for Exist and DoestNotExist

Reply:
I kept "IndexAlreadyExists" instead of `IndexDoesExists` is visually too close to `IndexDoesNotExists` and can lead to a hard time debugging.